java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -314,6 +320,13 @@ private static Object[] listCommands(BaseTransaction<?> transaction) {
                     new String[] {value3, value2}, // lrange(listKey1, 0, -2)
                     value3, // lpop(listKey1)
                     new String[] {value2, value1}, // lpopCount(listKey1, 2)
+                    5L, // lpush(listKey1, new String[] {value1, value1, value2, value3, value3})
+                    0L, // lpos(listKey1, value1)
+                    1L, // lpos(listKey1, value1, LPosOptions.builder().rank(2L).build())
+                    new Long[] {0L}, // lposCount(listKey1, value1, 1L)
+                    new Long[] {
+                        0L, 1L
+                    }, // lposCount(listKey1, value1, 0L, LPosOptions.builder().rank(2L).build())
```

Comment:
You can shorten the comment, so spotless won't split this line. It will be still understendable of what being tested.
```suggestion
                    new Long[] {0L, 1L}, // lposCount(listKey1, value1, 0L, LPosOptions.rank(2L))
```